pyproject.toml

```diff
@@ -101,7 +101,7 @@ strict = [
     "pydantic-settings==2.3.4",
     "pydantic==2.8.2",
     "pymatgen-analysis-defects==2024.7.19",
-    "pymatgen==2024.6.10",
+    "pymatgen@git+https://github.com/materialsproject/pymatgen.git@8e392947348c75220cfb79309fe8fc9745edc580",  # has to be adjusted with new pymatgen release
```

Comment:
I have fixed the pymatgen version to the merge that contains the changes in `CubicSupercellTransformation` in pymatgen.

Reply:
Thanks @QuantumChemist but we can't merge until a new pymatgen version has been released and this has been updated.

Reply:
Hi @utf , Ok when the new pymatgen release is out, I will also adjust it here :)